src/ert/gui/simulation/experiment_panel.py

```diff
@@ -297,3 +313,43 @@ def validationStatusChanged(self) -> None:
             self.run_button.text() == EXPERIMENT_READY_TO_RUN_BUTTON_MESSAGE
             and widget.isConfigurationValid()
         )
+
+    def populate_clipboard_debug_info(self) -> None:
+        output = ""
+        kv = {"**Platform**": "", ":-----": ":-----"}
+        kv["Date"] = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+        kv["OS"] = (
+            platform.system() + " " + platform.release() + " " + platform.machine()
+        )
+        kv["Hostname"] = platform.node()
+
+        for e in ["KOMODO_RELEASE"]:
+            if e in os.environ:
+                kv[e.capitalize()] = os.environ[e]
+
+        kv["Python Version"] = platform.python_version()
+
+        kv["**Ensemble**"] = ""
+        kv["Queue"] = self.config.queue_config.queue_system.name
+        kv["Simulation Mode"] = self.get_current_experiment_type().name()
```

Comment:
I am personally not fond of Title Casing, so I would write "Simulation mode" (and similarly for the others..)